data/inspect.csv

```diff
@@ -1084,7 +1084,7 @@ scidac.gov,http://scidac.gov,False,False,,False,False,False,False,False,False,Fa
 science.gov,http://www.science.gov,True,False,,True,False,False,False,False,False,False,,,False,False,False,False,False
 science360.gov,http://www.science360.gov,True,False,,False,False,False,False,True,True,False,,,False,False,False,False,False
 scienceaccelerator.gov,http://scienceaccelerator.gov,False,False,,False,False,False,False,False,False,False,,,False,False,False,True,True
-sciencebase.gov,https://www.sciencebase.gov,True,False,,True,True,False,True,False,False,False,,,False,False,False,False,False
+sciencebase.gov,http://www.sciencebase.gov,True,False,,False,False,False,False,False,False,False,,,False,False,False,False,False

```

Comment:
`sciencebase.gov` seem to be forcing it to me now. A note to ignore this next week if it shows up as forcing again.